man/rewritehead

```diff
@@ -70,3 +72,24 @@ RewriteRule ^cl_sampler_properties.html$ enums.html
 RewriteRule ^cl_svm_mem_flags.html$ enums.html
 RewriteRule ^memory_order.html$ enums.html
 RewriteRule ^memory_scope.html$ enums.html
+
+# Other aliases
+RewriteRule ^cl_khr_mipmap_image_writes.html$ cl_khr_mipmap_image.html
+RewriteRule ^cl_command_queue.html$ abstractDataTypes.html
+RewriteRule ^cl_context.html$ abstractDataTypes.html
+RewriteRule ^cl_device_id.html$ abstractDataTypes.html
+RewriteRule ^cl_event.html$ abstractDataTypes.html
+RewriteRule ^cl_kernel.html$ abstractDataTypes.html
+RewriteRule ^cl_mem.html$ abstractDataTypes.html
+RewriteRule ^cl_platform_id.html$ abstractDataTypes.html
+RewriteRule ^cl_program.html$ abstractDataTypes.html
+RewriteRule ^cl_sampler.html$ abstractDataTypes.html
+
+# Data types used in APIs, but not otherwise described in the specification
+RewriteRule ^cl_pipe_properties.html$ clCreatePipe.html
+RewriteRule ^cl_queue_properties.html$ clCreateCommandQueueWithProperties.html
+
+# Pages with no documentation at all, yet
+RewriteRule ^clCreateImageWithProperties.html$ deadLinks.html
```

Comment:
This should be able to go to the `clCreateImage` page, which also documents `clCreateImageWithProperties`.

https://www.khronos.org/registry/OpenCL/sdk/3.0/docs/man/html/clCreateImage.html

I'm surprised we don't have a similar issue with `clCreateBufferWithProperties`?

Reply:
I think there's a slight problem in the build toolchain - the refpage is tagged as an alias of clCreateImage but doesn't seem to be getting put into .htaccess as a redirect as it should. I'll take a look at that shortly.

Reply:
Fixed.